src/utils/validators.js

```diff
@@ -0,0 +1,21 @@
+const required = value => (value ? undefined : "Required");
+
+export const isPeselValid = pesel => {
```

Comment:
Please add a few unittests for checking if validators work are as expected.

Reply:
Just create separated file e.g. `validators.test.js` and call a function with parameters.

Reply:
Test file with pesel validator test cases have been added.